client/static/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Google</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Antic&family=Balsamiq+Sans:wght@700&family=Dancing+Script:wght@500&family=Lato:wght@300;400&family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
```

Comment:
(don't quote me on this) I think standard is to add this to your css file as it sort of falls under styling? could be wrong though

Reply:
Yur